runtime/Makefile.am

```diff
@@ -89,9 +90,13 @@ nodist_libcsmith_a_include_HEADERS = \
 BUILT_SOURCES = \
 	safe_math.h \
 	safe_math_macros.h \
-	safe_math_macros_notmp.h
+	safe_math_macros_notmp.h \
+	adapter.o \
+	adaptertracking.o
```

Comment:
This is just a quick note to say that I'm not ignoring this pull request and the mentions of my name :-). I've just been busy for the past few days. I hope to get to responding to the issues here soon.